client/state/selectors/get-gaining-registrar.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns gaining registrar data for a given domain, if we've successfully
+ * completed the transfer from our side.
+ *
+ * @param  {Object}  state       Global state tree
+ * @param  {string}  domain      Domain
+ * @return {Object}              Registrar object/record
+ */
+export default function getGainingRegistrar( state, domain ) {
+	return get( state, 'domains.transfer["' + domain + '"].selectedRegistrar', {} );
```

Comment:
Feel free to ignore this, but if you get comma coma, you can always use template literals here:
```
get( state, `domains.transfer['${ domain }'].selectedRegistrar`, {} );
```

Reply:
As a 3rd alternative, get accepts an array - `get( state, [ 'domains', 'transfer', domain, 'selectedRegistrar' ], {} )`.